specification.md

```diff
@@ -5989,6 +5990,60 @@ There are two places in the [workflow definition](#Workflow-Definition-Structure
 
 The `version` property must respect the [semantic versioning](https://semver.org/) guidelines.
 
+### Workflow Resources
```

Comment:
* Add example 
* Add roadmap 
* Check how we can use this in expressions that call expression functions, for example 
"condition": "${ fn:abc.isMinor }",

Reply:
Let me know when these things are added so i can review again.

Reply:
Will be done for tomorrow's meeting 😉